TaskWithNullptr/TaskWithNullptr.cpp

```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+#include <excpt.h>
+int main()
```

Comment:
Usually developers leave at least one line empty after headers. Please consider doing this in future.